hooks/useTokenList.tsx

```diff
@@ -34,18 +33,18 @@ export const useTokenList = () => {
         symbol: "WETH",
         address: network.wethAddress,
         decimals: 18,
-        name: "wETH",
+        name: "Wrapped ETH",
```

Comment:
Migrating this comment as a reminder to verify it's been resolved: https://github.com/ScopeLift/railgun-shielder/pull/46#discussion_r1082733840

Reply:
should be resolved, can you check it out?

Reply:
The addresses in `network.ts` look right! I'll mark this resolved